colors-and-fonts/index.css

```
@@ -61,7 +61,16 @@ body {
     flex-direction: column;
 }
 
-
+ul {
+    list-style-type: none;
+    margin: 5px;
+    padding: 25px;
+  }
+
+  ul :hover{
+      opacity: 50%;
+  }
+  
```

Would be good to have some specific classnames for those tags)
```suggestion
.nav {
    list-style-type: none;
    padding: 25px;
  }
  
  .nav-item {
   font-size: 16px;
   font-family: sans-serif;
   margin: 5px;
  }

  .nav-item:hover{
      opacity: 50%;
  }
  
```